### PR TITLE
Modified selector-append spec to test for multiple levels of input

### PR DESCRIPTION
**Pull request by onedayitwillmake**
Modified `selector-append` spec to confirm that more chained inputs were being caught

### CODE REVIEW

**Comment by onedayitwillmake** [comment]
Added spec for `simple-selectors` function

**Comment by onedayitwillmake** [comment]
@xzyfer Any chance for review?

File: spec/libsass-todo-issues/issue_963/selector-functions/simple_selectors/input.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.foo {
```
Comment (xzyfer) [inline spec/libsass-todo-issues/issue_963/selector-functions/simple_selectors/input.scss]: Should there be output for this case?

**Comment by xzyfer** [comment]
Rebase out c1b8a77e351021c400624a270e256b9ea0733482 and you good to go

**Comment by onedayitwillmake** [comment]
@xzyfer Added expected output, and rebased :)

**Comment by xzyfer** [comment]
Looks like you've merged rather than rebased. This has messed up this the commits in this PR (there are now 8 instead of 2).

If you're not familiar with how to rebase a PR please read this https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

**Comment by onedayitwillmake** [comment]
@xzyfer

I did debase which is why my commits were re-applied above master as shown above, however when I pushed it decided to auto merge my fork.

**Comment by xzyfer** [comment]
I don't feel comfortable merging this PR as is. I'll clean up the history down to the 1 or two commits that belong, unless you beat me to it.

**Comment by onedayitwillmake** [comment]
I will take care of it on my end, it's no problem 😁

I'll open another one tomorrow morning with just the new changes 🐣

**Pull request closed without merging**